godtools/Managers/TractManager.swift

```diff
@@ -16,57 +16,64 @@ class TractManager: NSObject {
 
 extension TractManager {
     
-    func loadResource(resource: String) -> (pages: [XMLIndexer], colors: TractColors) {
-        let manifestPath = resource + "-manifest"
-        let xmlData = loadXMLFile(manifestPath)
-        let manifest = xmlData["manifest"]
-        
-        let primaryColorString: String = (manifest.element?.attribute(by: "primary-color")?.text)!
-        let primaryTextColorString: String = (manifest.element?.attribute(by: "primary-text-color")?.text)!
-        let textColorString: String = (manifest.element?.attribute(by: "text-color")?.text)!
-        
+    func loadResource(resource: DownloadedResource, language: Language) -> (pages: [XMLIndexer], colors: TractColors) {
         var pages = [XMLIndexer]()
-        for child in manifest.children {
-            if child.element?.name == "page" {
-                let page = loadPage(child)
-                pages.append(page)
+        let tractColors = TractColors()
+        
+        let translation = resource.getTranslationForLanguage(language)
```

Comment:
I would do this instead:
````swift
guard let translation = resource.getTranslationForLanguage(language) else {
 return (pages, tractColors)
}
````
It's less indentation and keeps the "golden path".  See: https://github.com/raywenderlich/swift-style-guide#golden-path